src/lib/Libwin/accesinfo.c

```diff
@@ -106,9 +108,10 @@ create_secure_dacl(char *user, ACCESS_MASK mask, SID *owner_sid)
 				grp[k] = sid;
 				k++;
 			}
+		} else {
+			log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "failed to get group and user sid");
```

Comment:
Are these logs going to appear only in debug mode? I think we should log ALL errors in all logging levels